warehouse/models/mart/payments/fct_payments_rides_v2.sql

```diff
@@ -45,6 +45,11 @@
     filter_value = 'redwood-coast-transit',
     principals = ['serviceAccount:redwood-coast-transit@cal-itp-data-infra.iam.gserviceaccount.com']
 ) }}",
+" {{ create_row_access_policy(
+    filter_column = 'participant_id',
+    filter_value = 'atn',
+    principals = ['serviceAccount:atn@cal-itp-data-infra.iam.gserviceaccount.com']
```

Comment:
I don't see this service account, can you create it before merge? might error if it doesn't exist, I don't remember exactly

Reply:
Also I just realized we don't have the suffix `-payments-user` on all of these agency service accounts but it'd be useful especially on these abbreviated instance names

Reply:
Just created it, which also made me realize that the three-character name was too short for Google Cloud requirements, so it got a small rename in [9050782](https://github.com/cal-itp/data-infra/pull/2928/commits/90507826f8a0c8956316a3a1114f381a21bb293b)